### ISSUE TITLE
Compile error

### ISSUE THREAD

**Issue by Schaussi** [JoakimSoderberg/catcierge#10]
I tried to compily catcierge according to github readme.
Unfortunately, it does not find zmq library:

> Detected Raspberry Pi 2
> v0.6.3
> Catcierge version: 0.6.3
> CMake Error at /usr/share/cmake-3.0/Modules/FindPackageHandleStandardArgs.cmake:136 (message):
>   Could NOT find CZMQ (missing: CZMQ_LIBRARIES CZMQ_INCLUDE_DIRS)
> Call Stack (most recent call first):
>   /usr/share/cmake-3.0/Modules/FindPackageHandleStandardArgs.cmake:343 (_FPHSA_FAILURE_MESSAGE)
>   cmake/FindCZMQ.cmake:53 (find_package_handle_standard_args)
>   CMakeLists.txt:320 (find_package)

I tried to find/install packages like lib(c)zmq-dev, but with no success yet. I also manually compiled from zeromq, but the system can't find a czmq.h.
Did maybe a lib or file name change in a recent version?

Thank you

**Comment by JoakimSoderberg** [comment]
Yes Raspbian removed their CZMQ package in some release for some reason. Note that CZMQ is a simplified interface on top of the normal ZMQ, that's why installing that package did not help.

However, the CZMQ support is not vital, I only use that to broadcast events on my local network (to other raspberry PIs running this https://github.com/JoakimSoderberg/catcierge-cam). You can turn it off:

```bash
cmake -DWITH_ZMQ=OFF ..
```

Otherwise if you want support for it, you can build CZMQ:
https://github.com/zeromq/czmq

```bash
git clone git@github.com:zeromq/czmq.git
cd czmq
mkdir build
cd build
cmake ..
make
make install  # either this, or see below
```

or point to it explicitly when building:

```bash
cmake -DWITH_ZMQ=ON -DCZMQ_LIBRARIES=/path/to/czmq/src/libczmq.so -D CZMQ_INCLUDE_DIRS=/path/to/czmq/include
```

Are you planning to build your own cat door, or are you just curious? I'd love to hear how it goes if you do. No one else has built one that I know of :)

**Comment by Schaussi** [comment]
Thank you, it works now.
Yes, I am planning a similar project and I think your software could be very helpful.
As the rpi is placed at the cat door now, the next step would be to take a "good" snout pic, I hope I can use some cam motion detection package (I had also connected a motion detection sensor but it does not seem to work yet).
I will notify you of the project progress :)

**Comment by JoakimSoderberg** [comment]
ok but the motion detection is builtin into this code... What you need though is to have a white, preferably back lit background when taking the profile picture of the cat
